app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
Nếu không tìm thấy user thì sao?

Reply:
dạ! em da cho hienj thông báo

Reply:
tào lão, ri thì thấy user cũng hiện flash là user not found nè

1. dùng find_by thay vì find

2. thêm code xử lý khi user bị nil, kiểm tra user có bị nil ko mới add flash chứ cậu

Reply:
em da fix